docs/mp/grpc/client.adoc

```diff
@@ -28,261 +28,10 @@ include::{rootdir}/includes/mp.adoc[]
-
-And as with the server-side APIs, the Helidon MP gRPC client APIs support a number of different method signatures for each of the
-different gRPC method types.
-
-==== The Client Service Interface
-The next step is to produce an interface with the service methods that the client requires.
-
-For example, suppose we have a simple server side service that has a unary method to convert a string to uppercase.
-[source,java]
-.Simple gRPC Service
-----
-@ApplicationScoped
-@io.helidon.microprofile.grpc.core.Grpc
-public interface StringService {
-
-    @io.helidon.microprofile.grpc.core.Unary
-    public String upper(String s) {
-        return s == null ? null : s.toUpperCase();
-    }
-}
-----
-
-The service has been written using the Helidon MP APIs but could just as easily be a traditional gRPC Java service generated from
-Protobuf files. The client API is agnostic of the server side implementation, it only cares about the method types, the request
-and response types and the type of Marshaller used to serialize the request and response.
-
-To write a client for the StringService, all that is required is an interface.
-
-[source,java]
-.Simple gRPC Service
-----
-@ApplicationScoped
-@io.helidon.microprofile.grpc.core.Grpc
-public interface StringService {
-
-    @io.helidon.microprofile.grpc.core.Unary
-    public String upper(String s);
-}
-----
-
-There is no need to write any code to implement the client. The Helidon MP gRPC APIs will create a dynamic proxy for the interface
-using the information from the annotations and method signatures.
-
-The interface in the example above used the same method signature as the server but this does not have to be the case. It
-could have used any supported signature for a unary method. For example, it could just have easily been written using the standard
-unary method signature:
-
-[source,java]
-.Simple gRPC Service
-----
-@ApplicationScoped
-@io.helidon.microprofile.grpc.core.Grpc
-public interface StringService {
-
-    @io.helidon.microprofile.grpc.core.Unary
-    public void upper(String s, StreamObserver<String> response);
-}
-----
-
-We could also have made the client asynchronous by using one of the async method signatures:
-
-[source,java]
-.Simple gRPC Service
-----
-@ApplicationScoped
-@io.helidon.microprofile.grpc.core.Grpc
-public interface StringService {
-
-    @io.helidon.microprofile.grpc.core.Unary
-    public CompletableFuture<String> upper(String s);
-}
-----
-
-== Examples
-link:{helidon-github-tree-url}/examples/grpc/microprofile/basic-client[Basic gRPC Client] example demonstrates  a
-simple gRPC client that invokes services from deployed gRPC server applications provided in the
-link:{helidon-github-tree-url}/examples/grpc/microprofile/basic-server-implicit[Basic gRPC Server] and
-link:{helidon-github-tree-url}/examples/grpc/microprofile/metrics[gRPC Server metrics] examples.
+gRPC is temporarily removed in Helidon, please follow issue
```

Comment:
Do you want me to put this in Note format for more visibility? It's at the top of the page so I think its ok as-is.